### ISSUE TITLE
Out-ConsoleGridView -OutputMode Single - cannot select anything

### ISSUE THREAD

**Issue by hjorslev** [PowerShell/GraphicalTools#79]
# Steps to reproduce

```powershell
Get-Process | Out-ConsoleGridView -OutputMode Single
```
![image](https://user-images.githubusercontent.com/8019569/78059010-76426d00-7389-11ea-81f7-0cc9363fe84a.png)

# Expected behavior

An option to select one of the entries. If I run Out-GridView -OutputMode Single I can select one entry and have it returned back on the console:
![image](https://user-images.githubusercontent.com/8019569/78059153-adb11980-7389-11ea-86e3-0a4bdaed7cff.png)
![image](https://user-images.githubusercontent.com/8019569/78059175-b6095480-7389-11ea-97ea-072dd4120af7.png)

# Actual behavior

As far as I can see I have no option of selecting an item and have it returned when I run `Get-Process | Out-ConsoleGridView -OutputMode Single`. I have tried to select and pressed Space and Enter to select but I am still stuck in the menu and can only exit it by pressing F9.

# Environment data

$PSVersionTable:
```powershell
Name                           Value
----                           -----
PSVersion                      7.0.0
PSEdition                      Core
GitCommitId                    7.0.0
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0ΓÇª}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

```powershell

ModuleType Version    PreRelease Name                                ExportedCommands
---------- -------    ---------- ----                                ----------------
Binary     0.4.1                 Microsoft.PowerShell.ConsoleGuiTooΓÇª {Out-ConsoleGridView, ocgv}
```

**Comment by yanrez** [comment]
I hit the same problem. I'm using  "-OutputMode Single" and expected dialog to close on Enter with line I've selected.
Multi selection is also non-intuitive - not being able to close on Enter, nor using Insert for marking lines is very annoying behavior :(

**Comment by tig** [comment]
> I hit the same problem. I'm using "-OutputMode Single" and expected dialog to close on Enter with line I've selected.

This is fixed in a combination of

1) A PR to fix PR #81 - https://github.com/PowerShell/GraphicalTools/pull/86

2) A PR that introduces a status bar with ESC/ENTER support for Close/Accept. https://github.com/PowerShell/GraphicalTools/pull/86

> Multi selection is also non-intuitive - not being able to close on Enter, nor using Insert for marking lines is very annoying behavior :(

I've filed this against `Terminal.gui`: https://github.com/migueldeicaza/gui.cs/issues/385
